### ISSUE TITLE
The frame of a reaction doesn't have a constant thickness

### ISSUE THREAD

**Issue by elina2015** [status-im/status-desktop#5986]
# Bug Report
## Steps to reproduce
1. Add a reaction to 1-1 or any chat or community message
2. Check the frame of the reaction

#### Expected behavior
The frame of a reaction has a constant thickness
<img width="435" alt="image" src="https://user-images.githubusercontent.com/14942081/171760004-2b8b4fe9-dcf2-4a34-b149-ca5a9620c96c.png">

#### Actual behavior
The frame of a reaction doesn't have a constant thickness
### Additional Information

Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/master/2151/
Operating System: macOS Monterey 12.3 Beta (21E5227a)

**Comment by iurimatias** [comment]
@John-44 should it look like this?

![unknown (10)](https://user-images.githubusercontent.com/176720/183939844-63608868-a269-4557-a60c-f9ea8243d3f3.png)

**Comment by jrainville** [comment]
We can't find the desktop design :shrug:

**Comment by elina2015** [comment]
@igor-sirotin @iurimatias yep, it was the same for me - I didn't find the Design for this thing. I do believe it should be as we have it for the mobile app.

**Comment by John-44** [comment]
> @John-44 should it look like this?
> 
> ![unknown (10)](https://user-images.githubusercontent.com/176720/183939844-63608868-a269-4557-a60c-f9ea8243d3f3.png)

Yes exactly, is should look like the design shown here https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=1248%3A515

Reactions the user has made have a blue background, reactions the user hasn't made should have a grey background
